src/libraries/Common/src/System/IO/PathInternal.CaseSensitivity.cs

```diff
@@ -31,29 +31,13 @@ internal static StringComparison StringComparison
         /// <summary>
         /// Determines whether the file system is case sensitive.
         /// </summary>
-        /// <remarks>
-        /// Ideally we'd use something like pathconf with _PC_CASE_SENSITIVE, but that is non-portable,
-        /// not supported on Windows or Linux, etc. For now, this function creates a tmp file with capital letters
-        /// and then tests for its existence with lower-case letters.  This could return invalid results in corner
-        /// cases where, for example, different file systems are mounted with differing sensitivities.
-        /// </remarks>
         private static bool GetIsCaseSensitive()
```

Comment:
This should be `internal static bool IsCaseSensitive` property and `s_isCaseSensitive` should be deleted. It is not profitable to cache it in a static, and it prevents the linker constant propagation from kicking in.